includes/admin/class-wc-rest-payments-fraud-controller.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Class WC_REST_Payments_Fraud_Controller
+ *
+ * @package WooCommerce\Payments\Admin
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * REST controller for fraud mitigation.
+ */
+class WC_REST_Payments_Fraud_Controller extends WC_Payments_REST_Controller {
+	/**
+	 * Endpoint path.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'payments/fraud';
+
+	/**
+	 * Configure REST API routes.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/forter_token',
+			[
+				'methods'             => WP_REST_Server::CREATABLE,
+				'callback'            => [ $this, 'send_forter_token' ],
+				'permission_callback' => [ $this, 'check_permission' ],
+				'args'                => [
+					'token' => [
```

Comment:
I wonder if the token has some minimal string length or so, we might add the arg check here to be a bit more sure about the value we are passing to the fraud service

Reply:
That's a good point. The only thing they mention in API docs is the max length of 128 characters. Other than that, we don't know the allowed characters, or min length. Can you think of any additional check?

Reply:
Thank you for taking a second look at the docs!
I think in that case it's enough to check that the length is max 128 characters 👍